src/contracts/core/StrategyManager.sol

```diff
@@ -519,7 +502,7 @@ contract StrategyManager is
             } else {
                 if (queuedWithdrawal.strategies[i] == beaconChainETHStrategy){
                      //withdraw the beaconChainETH to the recipient
-                    _withdrawBeaconChainETH(queuedWithdrawal.depositor, recipient, queuedWithdrawal.shares[i]);
+                    eigenPodManager.withdrawRestakedBeaconChainETH(queuedWithdrawal.depositor, recipient, queuedWithdrawal.shares[i]);
```

Comment:
seems like a BUG -- now that we decrement on queuing, this would double-withdraw, right?

Reply:
I don't follow -  we only decrement withdrawableRestakedExecutionLayerGwei.  withdrawRestakedBeaconChainETH() just sends the actual ETH, doesn't decrement withdrawableRestakedExecutionLayerGwei

Reply:
ahhh mb, agreed it seems correct as-is